opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5-troubleshooting.adoc

```diff
@@ -0,0 +1,44 @@
+
+[[telemetryd-netflow5-troubleshooting]]
+===== NetFlow v5 Troubleshooting
+
+.Verify in Karaf Console if the `flows` feature is installed and started
+[source]
+----
+list | grep -i fail
```

Comment:
If the feature is not installed, this will show nothing - the same as if it were installed successfully.

I think using `feature:list | grep flows` would be a better approach here.